job-runner/chain/validate.go

```diff
@@ -16,7 +18,7 @@ func (e ErrInvalidChain) Error() string {
 }
 
 // Validate checks if a job chain is valid. It returns an error if it's not.
-func Validate(jobChain proto.JobChain) error {
+func Validate(jobChain proto.JobChain, new bool) error {
```

Comment:
I'd add an explanation of "new" in the comment.

Reply:
Updated.